### PR TITLE
bump to cedar@1.0.1 and use peerDependencies for arcgis-rest-js packages

### PR DESCRIPTION
**Pull request by tomwayson**

### CODE REVIEW

File: blueprints/ember-cli-cedar/index.js [hunk 1]
```diff
@@ -4,7 +4,11 @@ module.exports = {
```
Comment (jgravois) [inline blueprints/ember-cli-cedar/index.js]: does it matter that this file is named `index.js` and not `ember-cli-cedar.js`?

Comment (tomwayson) [reply blueprints/ember-cli-cedar/index.js]: [I _think_ so...](https://gist.github.com/kristianmandrup/ae3174217f68a6a51ed5#blueprints)

Comment (jgravois) [reply blueprints/ember-cli-cedar/index.js]: does that mean we need to rename the one in EAPS, torii, etc.?

Comment (tomwayson) [reply blueprints/ember-cli-cedar/index.js]: I don't think so? 🤷‍♂

**Pull request merged**